frontend-implicaction/src/app/job/services/job.service.ts

```diff
@@ -0,0 +1,25 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {ApiHttpService} from '../../core/services/api-http.service';
+import {ApiEndpointsService} from '../../core/services/api-endpoints.service';
+import {Pageable} from '../../shared/models/pageable';
+import {Observable} from 'rxjs';
+import {JobPosting} from '../../shared/models/job-posting';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class JobService {
+
+  constructor(
+    private http: HttpClient,
+    private apiHttpService: ApiHttpService,
+    private apiEndpointsService: ApiEndpointsService
```

Comment:
```suggestion
    private http: HttpClient,
    private apiEndpointsService: ApiEndpointsService
```
On n'utilise plus ApiHttpService parce que pas de réel apport pour nous, il faudrait plutôt passer par HttpClient. Du coup il faut supprimer cette variable